pyproject.toml

```diff
@@ -27,7 +27,7 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = "^3.6.2"
```

Comment:
Why would this be added back?

Reply:
The version of `black` required for GHA wouldn't build until the dependency was raised to `3.6.2`. It was complaining that 3.6 would not satisfy the `black requirement`

```
The current project's Python requirement (>=3.6,<4.0) is not compatible with some of the required packages Python requirement:
    - black requires Python >=3.6.2, so it will not be satisfied for Python >=3.6,<3.6.2
    - black requires Python >=3.6.2, so it will not be satisfied for Python >=3.6,<3.6.2
    - black requires Python >=3.6.2, so it will not be satisfied for Python >=3.6,<3.6.2
  
  Because no versions of black match >21.10b0,<21.11b0 || >21.11b0,<21.11b1 || >21.11b1,<22.0
   and black (21.10b0) requires Python >=3.6.2, black is forbidden.
  And because black (21.11b0) requires Python >=3.6.2, black is forbidden.
  So, because black (21.11b1) requires Python >=3.6.2
   and netutils depends on black (^21.10b0), version solving failed.
```

Reply:
But this is for development only, so I do not think it matters. Development should be done in 3.6.2 but the library should be fine.

Reply:
Thinking about it, @qduk can you document this in the contrib guide?